docs/troubleshooting.md

```diff
@@ -56,3 +57,12 @@ clouds:
     cacert: /etc/certs/cacert
     verify: false
 ```
+
+## Fails in creating floating IP during cluster creation.
+
+If you encounter `rule:create_floatingip and rule:create_floatingip:floating_ip_address is disallowed by policy` when create floating ip, check with your openstack administrator, you need to be authorized to perform those actions, see (issue 572)[https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/572) for more detailed information.
```

Comment:
Link is not working. Please reverse [] and ().

Reply:
done